content/v2.0/query-data/common-queries/median.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Query median values
+seotitle: Query median values in Flux
+list_title: Median
+description: >
+  Use the `median()` function to return all values within the `0.5` quantile
+  (50th percentile) or the median value of input data.
+weight: 202
+menu:
+  v2_0:
+    parent: Common queries
+    name: Median
+v2.0/tags: [query, median]
+related:
+  - /v2.0/query-data/common-queries/percentile-quantile/
+---
+
+Use the [`median()` function](/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/median/)
+to return all values within the `0.5` quantile (50th percentile) or the median value of input data.
+
+## Median calculation methods
+Select from the following methods for calculating the median:
```

Comment:
##Calculate the median
Select one of the following methods to calculate the median